### PR TITLE
Complementary palette in filters

### PR DESCRIPTION
**Pull request by scavnickyj**
- introduce complementary palette to date and attribute filters

JIRA: BB-2818

---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2590/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4220/ : SUCCESS in 14m 58s

**Comment by scavnickyj** [comment]
extended test

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/694/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2591/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/694/ : SUCCESS in 21m 13s

File: libs/sdk-ui-kit/styles/scss/header.scss [hunk 1]
```diff
@@ -23,7 +23,7 @@ $button-normal-active-shadow: transparentize($button-normal-active-border-color,
     height: $gd-header-height;
     padding: 0 0 0 20px;
     box-shadow: 0 1px 3px rgba(0, 0, 0, 0.35);
-    color: $gd-color-text-light;
+    color: #fff;
```
Comment (yanlipnican) [inline libs/sdk-ui-kit/styles/scss/header.scss:26]: this is intentional?

File: libs/sdk-ui-kit/styles/scss/datepicker.scss [hunk 2]
```diff
@@ -178,7 +178,7 @@
 
     .DayPicker-Caption {
         display: table-caption;
-        color: #000;
+        color: $gd-color-dark;
```
Comment (yanlipnican) [inline libs/sdk-ui-kit/styles/scss/datepicker.scss:181]: this is intentional?

File: libs/sdk-ui-filters/styles/scss/components/SelectMenu.scss [hunk 3]
```diff
@@ -13,10 +13,10 @@
     line-height: 1.4rem;
     border-radius: 3px;
     font-family: $gd-font-primary;
-    background-color: #fff;
+    background-color: $gd-color-light;
```
Comment (yanlipnican) [inline libs/sdk-ui-filters/styles/scss/components/SelectMenu.scss:16]: this is intentional?

File: libs/sdk-ui-filters/styles/scss/components/DateRangePicker.scss [hunk 4]
```diff
@@ -166,7 +167,7 @@ $gd-day-picker-width: 268px;
             font-weight: 400;
             font-size: 17px;
             text-align: center;
-            color: #000;
+            color: $gd-color-dark;
```
Comment (yanlipnican) [inline libs/sdk-ui-filters/styles/scss/components/DateRangePicker.scss:170]: this is intentional?

**Review by yanlipnican — APPROVED** [review]

File: libs/sdk-ui-filters/styles/scss/components/DateRangePicker.scss (hunk 4 above)
Comment (scavnickyj) [reply libs/sdk-ui-filters/styles/scss/components/DateRangePicker.scss:170]: yes

**Comment by yanlipnican** [comment]
LGTM :rocket:

File: libs/sdk-ui-kit/styles/scss/header.scss (hunk 1 above)
Comment (scavnickyj) [reply libs/sdk-ui-kit/styles/scss/header.scss:26]: yes, the header should not be affected by complementary palette

File: libs/sdk-ui-kit/styles/scss/datepicker.scss (hunk 2 above)
Comment (scavnickyj) [reply libs/sdk-ui-kit/styles/scss/datepicker.scss:181]: yes

File: libs/sdk-ui-filters/styles/scss/components/SelectMenu.scss (hunk 3 above)
Comment (scavnickyj) [reply libs/sdk-ui-filters/styles/scss/components/SelectMenu.scss:16]: yes

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1074

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/777/ : SUCCESS in 28s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1040/ : SUCCESS in 22s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4223/ : SUCCESS in 15m 26s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-el7/140/ : SUCCESS in 10m 23s (non-voting)

**Pull request merged**